service/src/main/resources/db/migration/V72__messages.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE message_account (
+    id uuid PRIMARY KEY DEFAULT ext.uuid_generate_v1mc(),
+    created timestamp with time zone DEFAULT now() NOT NULL,
+    updated timestamp with time zone DEFAULT now() NOT NULL,
+    daycare_group_id uuid references daycare_group(id),
+    employee_id uuid references employee(id),
+    person_id uuid references person(id)
+);
+CREATE TRIGGER set_timestamp BEFORE UPDATE ON message_account FOR EACH ROW EXECUTE PROCEDURE trigger_refresh_updated();
+
+ALTER TABLE message_account ADD CONSTRAINT message_account_created_for_fk CHECK (
+    num_nonnulls(daycare_group_id, employee_id, person_id) = 1
+);
+ALTER TABLE message_account ADD CONSTRAINT message_account_daycare_group_uniq UNIQUE (daycare_group_id);
+ALTER TABLE message_account ADD CONSTRAINT message_account_employee_uniq UNIQUE (employee_id);
+ALTER TABLE message_account ADD CONSTRAINT message_account_person_uniq UNIQUE (person_id);
+
+INSERT INTO message_account (daycare_group_id)  SELECT id from daycare_group;
+INSERT INTO message_account (employee_id) SELECT id from employee;
+INSERT INTO message_account (person_id) SELECT id from person;
+
+CREATE INDEX idx$message_account_daycare_group_id ON message_account (daycare_group_id);
+CREATE INDEX idx$message_account_employee_id ON message_account (employee_id);
+CREATE INDEX idx$message_account_person_id ON message_account (person_id);
+
+CREATE TABLE message (
+    id uuid PRIMARY KEY DEFAULT ext.uuid_generate_v1mc(),
+    created timestamp with time zone DEFAULT now() NOT NULL,
+    updated timestamp with time zone DEFAULT now() NOT NULL,
+    title text,
```

Comment:
should this be not null? empty string is still possible

Reply:
The UX design indicated that thread replies do not have titles so I figured null would be semantically more correct than empty string.